README.md

```diff
@@ -94,13 +94,16 @@ Copy your converted token and paste it in `/src/portfolio.js`
 
 ### To Change website content go to `/src/portfolio.js` & modify it as per your need.
 
+For adding emoji 😃 into the texts in Protofolio.js, you should use `emoji()` function and pass the text you need to add as an argument. This would help in uniform rendering of Emoji. 
+
 ```javascript
 /* Change this file to get your Personal Porfolio */
 
-const gretting = {
-  /* Your Summary And Gretting Section */
-  title: "Hi all 👋 I'm Saad",
-  subTitle: "A passionate Full Stack Software Developer 🚀.
+const greeting = {
+  /* Your Summary And Greeting Section */
+  title: "Hi all I'm Saad",
+  subTitle:
+    emoji("A passionate Full Stack Software Developer 🚀"),
```

Comment:
Could you move this to one line for consistency?
Like,
subTitle: emoji("A passionate Full Stack Software Developer 🚀"),